schema/products/ibu10m.json

```diff
@@ -0,0 +1,309 @@
+            "end_datetime",
+            "created",
+            "providers",
+            "proj:epsg",
+            "proj:bbox",
+            "proj:shape"
+          ],
+          "properties": {
+            "providers": {
+              "const": [
+                {
+                  "name": "Copernicus Land Monitoring Service",
+                  "description": "The Copernicus Land Monitoring Service provides geographical information on land cover and its changes, land use, ground motions, vegetation state, water cycle and Earth's surface energy variables to a broad range of users in Europe and across the World in the field of environmental terrestrial applications.",
+                  "roles": ["licensor", "host"],
+                  "url": "https://land.copernicus.eu"
+                }
+              ]
+            }
+          }
+        },
+        "collection": {
+          "const": "imperviousness-built-up-10m"
+        },
+        "links": {
+          "type": "array",
+          "allOf": [
+            {
+              "contains": {
+                "const": {
+                  "rel": "license",
+                  "href": "https://land.copernicus.eu/en/data-policy"
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "self"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "root"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "parent"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "collection"
+                  }
+                }
+              }
+            }
+          ]
+        },
+        "assets": {
+          "type": "object",
+          "minProperties": 3,
+          "maxProperties": 3,
+          "propertyNames": {
+            "pattern": "^builtup_map"
+          },
+          "additionalProperties": {
+            "type": "object",
+            "properties": {
+              "href": {
+                "type": "string",
+                "pattern": ".tif$|.xml$|.clr$|.ovr$|.cpg$|.dbf$|.tfw$|.lyr$|.txt$"
+              }
+            }
+          }
+        }
+      }
+    },
+    {
+      "$comment": "This is the schema for Imperviousness Built-Up 10m Collection",
+      "type": "object",
+      "required": [
+        "stac_version",
+        "stac_extension",
```

Comment:
I made a mistake in the sample file. This should be `stac_extensions` according to the [collection-spec](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#collection-fields). Please update this and the `stac_extensions` field in the [`imperviousness-built-up-10m.json`](https://github.com/eea/clms-stac/blob/develop/stacs/imperviousness-built-up-10m/imperviousness-built-up-10m.json#L4).